generic/tests/kdump.py

```diff
@@ -147,15 +150,22 @@ def crash_test(test, vm, vcpu, crash_cmd, timeout):
     vmcore_rm_cmd = vm.params.get("vmcore_rm_cmd", "rm -rf %s/*")
     vmcore_rm_cmd = vmcore_rm_cmd % vmcore_path
     kdump_restart_cmd = vm.params.get("kdump_restart_cmd",
-                                      "service kdump restart")
+                                      "systemctl restart kdump.service")
     kdump_status_cmd = vm.params.get("kdump_status_cmd",
                                      "systemctl status kdump.service")
+    kdump_propagate_cmd = vm.params.get("kdump_propagate_cmd",
+                                        'kdumpctl propagate')
 
     session = vm.wait_for_login(timeout=timeout)
-
     logging.info("Delete the vmcore file.")
     if kdump_method == "ssh":
+        output = session.cmd("cat /root/.ssh/kdump_id_rsa.pub")
         process.run(vmcore_rm_cmd, shell=True)
+        process.run("cat /dev/null > /root/.ssh/authorized_keys",
```

Comment:
Better backup and restore it after the test.

Reply:
Ok,thanks.